pkg/bme680_driver/README.md

```diff
@@ -0,0 +1,26 @@
+# bme680_driver
+
+## Introduction
+[bme680](https://github.com/BoschSensortec/BME680_driver) is an I2C/SPI API for BME680 sensor.
+
+The library is written and maintained by Bosch Sensortec, it is platform independent, as long as the right drivers are available for the given MCU.
+
+In addition, this driver can use floating point if available on your MCU. By default, this package do not use it.
+
+## Usage
+Add `USEPKG += bme680_driver` in your Makefile and `#include "bme680_hal.h"` to your code. Refer to the code documentation for more information on the API.
```

Comment:
Is this really necessary to add `USEPKG += bme680_driver`, you already include it in `drivers/Makefile.dep` when module `bme680` is used?

Reply:
Please remove `USEPKG += bme680_driver` from documentation. It isn't necessary to add it to the application's makefile.

Even more, if you move
```
#include "bme680_hal.h"
#include "bme680_defs.h"
```
to `bme680.h`, the application has not to include them. Then it is enough to include `bme680.h` and `bme680_params.h` by the application. This makes the usage easier.
```suggestion
Refer to the code documentation for more information on the API.
```